src/lib/api/ipfs.ts

```diff
@@ -51,8 +50,9 @@ export const pinFile = async (
 ) => {
   const formData = new FormData()
   formData.append('file', image)
-  const res = await axios.post<IpfsPinFileResponse>(
-    '/api/ipfs/pinFile',
+
+  const res = await axios.post<InfuraPinResponse>(
+    'https://api.juicebox.money/api/ipfs/file',
```

Comment:
instead of POSTing to our old Vercel endpoint (now deleted), we POST to https://api.juicebox.money